jerry-core/debugger/debugger.h

```diff
@@ -23,6 +23,11 @@
 
 /* JerryScript debugger protocol is a simplified version of RFC-6455 (WebSockets). */
 
+/**
+ * JerryScript debugger protocol version.
+ */
+#define JERRY_DEBUGGER_VERSION (4)
```

Comment:
Let's start with version 0.

Reply:
I would start with 1 if that's not a problem for you (zero is not really a version number imho).